### PR TITLE
APIGOV-18306 - Update page in Connect Azure Gateway “agent-variables”

### PR DESCRIPTION
**Pull request by sbolosan**
Automatically generated by Netlify CMS

### CODE REVIEW

**Comment by sbolosan** [comment]
@jcollins-axway @kflan-io @dfeldick @vivekschauhan - please review when you get the chance por favor

**Review by dfeldick — APPROVED** [review]

**Review by kflan-io — APPROVED** [review]

**Review by cbordier — APPROVED** [review]
I approve the changes

File: content/en/docs/central/connect-azure-gateway/agent-variables.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ You can extend the previous minimum variable list with the following variables.
...
 | AZURE_RESOURCEGROUPNAME                                                | The container name that holds resources.                                                                                                                                                                                                                                                                                                                                                                                             |
 | AZURE_APIMSERVICENAME                            | The container that holds the API you want the agent to discover.                                                                                                   |
+| AZURE_PUSHTAGS                            | When set to TRUE, the Azure API tags to be pushed to AMPLIFY Central.                                                                                                   |
```
Comment (lbadenhop) [inline content/en/docs/central/connect-azure-gateway/agent-variables.md:36]:
```suggestion
| AZURE_PUSHTAGS                            | When set to TRUE, the Azure API tags will be pushed to AMPLIFY Central.                                                                                                   |
```

**Review by lbadenhop — APPROVED** [review]

File: content/en/docs/central/connect-azure-gateway/agent-variables.md (hunk 1 above)
Comment (sbolosan) [reply content/en/docs/central/connect-azure-gateway/agent-variables.md]: changed...thanks

**Pull request merged**